target/linux/ath79/dts/qca9558_sitecom_wlr-8100.dts

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	model = "Sitecom WLR-8100 (X8 AC1750)";
+	compatible = "sitecom,wlr-8100", "qca,qca9558";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &led_status_amber;
+		led-failsafe = &led_status_amber;
+		led-running = &led_status_amber;
+		led-upgrade = &led_status_amber;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wifi2g {
```

Comment:
On could name these to include both uses: wifi2g_rfkill and wifi5g_restart

Reply:
Fixed, thanks